### ISSUE TITLE
Style/Color Refresh

### ISSUE THREAD

**Issue by JonDum** [ractivejs/ractivejs.github.io#16]
![style-guide](https://cloud.githubusercontent.com/assets/191920/22801395/df272ef2-eec9-11e6-94cc-012e97ccafa4.png)

Had a little time to play around with it. Wanted to keep Ractive's "green" theme going, but make it a little more vibrant and exciting... maybe too much? Not sure yet.

**Comment by fskreuz** [comment]
> My designer buddies have forbidden me from trying to be aesthetical (box shadows and rounded corners are not primary design elements, I'm told 😁)
> https://github.com/ractivejs/ractive/issues/2857#issuecomment-277885243

😂 Also, the neon is hard on the eyes. 

I like the font and the font color though. Reminds me of the old Node.js color scheme (green on grey). The green tinge on the font is a nice departure from the usual `#222` and `#333` font colors of CSS frameworks.

**Comment by fskreuz** [comment]
Also, Ractive's name came from... a book http://www.ifwiki.org/index.php/Ractive.

**Comment by evs-chris** [comment]
I like all of that, except maybe the gradient start. Perhaps it could swing toward yellow rather than blue?

**Comment by JonDum** [comment]
Here's one that's toned down. Green-blue -> Green

<img width="667" alt="screen shot 2017-02-09 at 9 34 32 pm" src="https://cloud.githubusercontent.com/assets/191920/22814299/9c877d7c-ef0f-11e6-8c5c-0c063ef208f3.png">

Here's from a yellowish hue.

<img width="669" alt="screen shot 2017-02-09 at 9 36 27 pm" src="https://cloud.githubusercontent.com/assets/191920/22814324/d8fef550-ef0f-11e6-93ae-b34b8cdc7f7e.png">

I'm liking the yellowish, nice suggestion Chris!

I started playing around with what the homepage might look like, but honestly, it takes me so long to do stuff in Affinity Designer that it might be faster if I just start hacking it together in code :rofl:

**Comment by fskreuz** [comment]
Per Gitter conversation, new docs is:

- Perceivably too wordy.
- Too docs-like.
- Not as interactive as the old site.
- Other stuff (see convo starting at https://gitter.im/ractivejs/ractive?at=59034f22d32c6f2f094cb6ef)

---

References to the fonts used in old site

- https://fonts.google.com/specimen/Voltaire
- https://fonts.google.com/specimen/Source+Sans+Pro

---

Framework docs that I personally find nice

- https://infernojs.org/docs/api/inferno
- https://vuejs.org/v2/api/
- http://redux.js.org/

**Comment by andreujuanc** [comment]
@JonDum I liked bit more the one with yellowish hue. Actually I was thinking something like that when I first saw the first images.

**Comment by fskreuz** [comment]
As much as a like to consolidate docs first, the opportunity of doing a design PoC was too tempting. 😈 Plus words are too clumsy to convey the idea. Here's some samples of what I imagine the design would be. 

The home page design is just a refresh of the original site's home page. Green highlights on white is the name of the game. No excessive lines, boxes, and borders to remove visual traffic. If you want to be snappy, look snappy. 

Only 3 colors: The original green (#729D34), text color grey (#555) and a highlights grey (#fafafa). Fireworks doesn't export #fafafa very well, so the examples code and result blocks are a bit darker than what they should be. Fonts are still Voltaire and Source Sans Pro.

The front page will contain the stuff from the Examples page. This will also eliminate the Examples page entirely since it will become redundant. But hey! One less page to manage, one less page to read. Only the shorter examples will be put in though.

The original site had a support section on the lower right. Since there's a dedicated support page (which will probably turn into an FAQ page with all the common questions I get from Github/Stack Overflow), I omitted it from the design.

![ractive-home-desktop](https://cloud.githubusercontent.com/assets/1582932/25558792/11977912-2cfc-11e7-8cf7-48670ff51120.png)

---

On mobile, bootstrap-style collapse for the menu. The examples will be selected from a drop-down, only the code will show by default, and the "View Result" will toggle between code and result. I put in "Get Ractive.js" for the fun of it, but it will be removed since it makes no sense to get code on mobile. Base font size drops from 18px to 16px, everything else should be relative in rems.

![ractive-home-mobile](https://cloud.githubusercontent.com/assets/1582932/25558795/119bdb56-2cfc-11e7-95f4-30b6ad1fc0f8.png)

---

For the docs, not any different from the current js.org version, but less lines, blocks and unnecessary stuff. The red parts will be colorized by a syntax highlighter. Got lazy and did not put the grey background on the code. However, highlighted code on white background isn't that bad. Reminds me of VS Code using its light theme.

The sidebar will not stick this time. Dealing with sticky sidebars is a hassle when it comes to positioning, z-index fighting, scrolling and stuff.

![ractive-docs-desktop](https://cloud.githubusercontent.com/assets/1582932/25558793/1197c20a-2cfc-11e7-870d-c5fbee9e00ce.png)

---

Mobile pages gets rid of the sidebar entirely. Main menu still stays the same.

![ractive-docs-mobile](https://cloud.githubusercontent.com/assets/1582932/25558794/119956ba-2cfc-11e7-86ec-7cdf90567843.png)

---

Comments? Suggestions? Outrageous reactions? 😄

**Comment by andreujuanc** [comment]
@fskreuz nice! love it. Maybe the Get ractive stuff on top?

![image](https://cloud.githubusercontent.com/assets/360052/25559526/76f8e486-2d3c-11e7-8b38-56f580912cd9.png)

**Comment by fskreuz** [comment]
Yup, thought about putting it back there. It looked weird at first which is why I put it below. Was thinking about the flow of content being "What is this Ractive you speak of?" to "What can it do?" to "Where can I get this awesome library". 😄

**Comment by kouts** [comment]
Nice one @fskreuz ! 
+1 for the sidebar, it's always nice to see all that's available.

**Comment by dagnelies** [comment]
Nice one @fskreuz, I like that theme suggestion a lot too!

**Comment by fskreuz** [comment]
I think I'll call this one done. Fresh coat of paint is now applied, https://ractive.js.org/ Feel free to leave feedback though. 😄
